test/streaming/testResultStringifyStream.test.ts

```diff
@@ -155,11 +182,12 @@ describe('TestResultStringifyStream', () => {
   });
   it('should transform TestResult into a JSON string', (done) => {
```

Comment:
Nit - more detailed test case name 🙂

```suggestion
  it('should transform TestResult into a JSON string with tests and coverage both present', (done) => {
```